src/driver.cpp

```diff
@@ -46,7 +50,11 @@ int Driver::parse()
   if (debug_) {
     parser.set_debug_level(1);
   }
-  yy_scan_string(Log::get().get_source().c_str(), scanner);
+  auto &source = Log::get().get_source();
+  yy_scan_string_with_size(source.c_str(),
+                           std::max(source.size(),
+                                    static_cast<size_t>(MIN_YY_BUF_SIZE)),
```

Comment:
If you need this to be `size_t`, maybe just declare it as `static constexpr size_t`, then? Instead of a macro